general/administration/reverse-proxy.md

```diff
@@ -197,13 +198,8 @@ server {
 
     # Jellyfin
     location /jellyfin {
-        return 302 $scheme://$host/jellyfin/;
-    }
-
-    location /jellyfin/ {
         # Proxy main Jellyfin traffic
-        # The / at the end is significant.
-        proxy_pass http://SERVER_IP_ADDRESS:8096/;
+        proxy_pass http://SERVER_IP_ADDRESS:8096/jellyfin/;
```

Comment:
This doesn't feel correct. It can have unintended side effects like passing to a JF server a url like `DOMAIN/jellyfin2/foo` -> `INTERNAL/jellyfin/2/foo`